kiwi/schema/kiwi.rnc

```diff
@@ -870,9 +870,8 @@ div {
     k.repository.type.attribute =
         ## Type of repository
         attribute type {
-        "apt-deb" | "apt-rpm" | "deb-dir" | "mirrors" | "red-carpet" |
-        "rpm-dir" | "rpm-md"  | "slack-site" | "up2date-mirrors" | "urpmi" |
-        "yast2"
+        "apt-deb" | "apt-rpm" | "deb-dir" | "mirrors" | "rpm-dir" |
+        "rpm-md"  | "yast2"
```

Comment:
We should also drop `yast2`. From YaST2 and Zypper's point of view, it only accepts `rpm-md` nowadays, and all SUSE products supported offer `rpm-md` anyway.

Reply:
correct at least sle12 which is the oldest target supported by the kiwi-ng code base this is already the case. So yes yast2 can be deleted too

Reply:
I'll drop that one too.

Reply:
Done